mailpoet/package.json

```diff
@@ -146,10 +146,12 @@
     "@types/wordpress__edit-post": "^7.5.6",
     "@types/wordpress__editor": "^13.6.7",
     "@types/wordpress__media-utils": "^4.14.4",
+    "@types/wordpress__notices": "^3.27.6",
```

Comment:
Do we need to add the package? I see that @wordpress/notices contain build-types. AFAIK if a package provides own build types the type definitions from @types and basically ignored. I also see you added a type for Notice. I tried removing the package locally and QA checks  for js `./do qa:lint-javascript` are still passing. Am I missing something or can we remove the package?

Reply:
I added this trying to get the `Notice` type working but in the end I had to duplicate it. I can remove it, it wasn't useful.